DEVELOPING.md

```diff
@@ -24,6 +24,70 @@ To do a full clean run:
 yarn clean_all
 ```
 
+## Debugging
+
+Adding following options to your `.bazelrc` file to add support for debugging NodeJS tests and to make the build and test
+output more verbose.
+
+```
+# Support for debugging NodeJS tests
```

Comment:
maybe just point to common.bazelrc so we don't have to duplicate it in markdown

Reply:
👍